src/openforms/submissions/admin.py

```diff
@@ -17,6 +18,7 @@
 )
 from openforms.logging.models import TimelineLogProxy
 from openforms.payments.models import SubmissionPayment
+from openforms.submissions.views import LogsEvaluatedLogicView
```

Comment:
I'd move this view implementation to `openforms.submissions.admin_views`. Later, we should restructure the admin code so that it's a folder `admin/` with python modules inside, similar to `openforms.forms.admin`.

Splitting between `views.py` and `admin_views.py` makes it more obvious for developers in which context these views are used (actual end-users vs. staff users)